sdk/storage/Directory.Build.props

```diff
@@ -0,0 +1,87 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+  Import our Directory.Build.props and /eng infrastructure
+  -->
+  <PropertyGroup>
+    <!--
+    We're currently on 1.0.0-preview.X and we'll share this version number
+    across all Storage projects until the need to version independently arises.
+    See /eng/Versioning.props for more details.
+    -->
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
So this is the single place we edit our version number?

Reply:
Yes it should be the single place.

Reply:
Yes.